cql/ScreeningAverageRiskLibrary.cql

```diff
@@ -93,10 +93,13 @@ define HasHighGradePreCancerCervicalLesion:
 define HasCervicalCancerDiagnoses:
   Exists(Dash.CervicalCancerDiagnoses)
 
-define Cin2OrCin3HistologyResults:
+define HighGradeOrCancerHistologyResults:
```

Comment:
This was something I went back and forth on. I settled on keeping the logic in each individual logic path where it is used (1) in order to align with how it is structured in the L2 and (2) since the identical expressions come from different guidelines, an L2 change for one may not necessarily imply an L2 change for another.

For now, they are identical, so I agree it makes sense to abstract the logic to the Dashboard to avoid having to repeat it in LP2, LP4, and LP5.